easybuild/easyconfigs/n/NCO/NCO-4.6.0-foss-2016a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'NCO'
+version = '4.6.0'
+
+homepage = "http://nco.sourceforge.net"
+description = """manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://nco.sourceforge.net/src']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('UDUNITS', '2.2.20'),
+    ('expat', '2.1.0'),
+    ('ANTLR', '2.7.7', '-Python-2.7.11'),
+    ('GSL', '2.1'),
+    ('netCDF', '4.4.0'),
```

Comment:
existing NCO easyconfigs also list `Bison`, `flex`, `libdap`, `netcdf4-python` and `scipy` as dependencies?